portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,82 @@
+function loadInFact(index, fact_index, isTruth) {
```

Comment:
In this parameter list and the rest of this file.. the style guide calls for camelCase in all variable names and method names (which it looks like you're already doing for method names) =)

So, factIndex here.  And below, we want things like firstTruthValue and secondTruthValue.

Reply:
Change parameters to camelcase